sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/SparkGetFunctionsOperation.scala

```diff
@@ -91,7 +88,7 @@ private[hive] class SparkGetFunctionsOperation(
               DEFAULT_HIVE_CATALOG, // FUNCTION_CAT
               db, // FUNCTION_SCHEM
               funcIdentifier.funcName, // FUNCTION_NAME
-              info.getUsage, // REMARKS
+              "    Usage:\n      " + info.getUsage.trim + "\n" + info.getExtended, // REMARKS
```

Comment:
```s"Usage: ${info.getUsage}\nExtended Usage:${info.getExtended}"```?  In order to match `DescribeFunctionCommand`: https://github.com/apache/spark/blob/7048fff2304bd44c3c6b7e57a485200a8959203d/sql/core/src/main/scala/org/apache/spark/sql/execution/command/functions.scala#L144-L148

Reply:
LGTM, I will follow it.